components/Nav/Navbar/index.tsx

```diff
@@ -171,9 +171,8 @@ const DropdownLogo = styled(({ className }) => {
     }
 
     > .dropdown-menu {
-        opacity: 0;
         position: absolute;
-        display: block;
+        display: none;
```

Comment:
Cant we increase/decrease z-index on the nav to fix this without using display none??

Reply:
Will fix